cpp/style.rst

```diff
@@ -195,17 +195,17 @@ The capitalization rule for class names should be all words in the name capitali
 
 .. _style-guide-cpp-3-2:
 
-3-2. Variable names MUST be in mixed case starting with lower case.
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+3-2. Variable names MUST be camelCase or snake_case.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. code-block:: cpp
 
    int lineWidth;
+   // OR, in primarily new code:
+   int line_width;
 
-Common practice in the C++ development community.
-Makes variables easy to distinguish from types, and effectively resolves potential naming collision as in the declaration Line line.
-Keep variable names balanced between short and longer, more meaningful.
-Use 8 to 20 characters as a guideline (excluding integer loop counters which may be as little as 1 character).
```

Comment:
Is it worth keeping this suggestion?

Reply:
I've restored the last two bullets, as those are indeed suggestions; I'd missed the distinction in my first read-through of these comments.